futures-executor/src/lib.rs

```diff
@@ -2,17 +2,13 @@
 
 #![feature(pin, arbitrary_self_types, futures_api)]
 
-#![no_std]
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Rustdoc doesn't find the links in the doc comments from futures-util otherwise.

Reply:
Interesting!